bigtable/client/client_options_test.cc

```diff
@@ -18,15 +18,21 @@
 
 TEST(ClientOptionsTest, ClientOptionsDefaultSettings) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
-  EXPECT_EQ("bigtable.googleapis.com", client_options_object.endpoint());
+  EXPECT_EQ("bigtable.googleapis.com", client_options_object.data_endpoint());
+  EXPECT_EQ("bigtableadmin.googleapis.com",
+            client_options_object.admin_endpoint());
   EXPECT_EQ(typeid(grpc::GoogleDefaultCredentials()),
             typeid(client_options_object.credentials()));
 }
 
 TEST(ClientOptionsTest, ClientOptionsCustomEndpoint) {
+  // TODO(#23) - setenv() is a Unix specific call ...
   setenv("BIGTABLE_EMULATOR_HOST", "testendpoint.googleapis.com", 1);
```

Comment:
Have to be really careful here: if this test ends too early (due to `ASSERT_*` being used instead of `EXPECT_*` at some point in the future, for example, this env var will stick around, affecting other tests, and making it quite hard to debug.

A way to guarantee that this happens is to create a custom test fixture class, where the `SetUp()` method does the `setenv()` call and `TearDown()` does the `unsetenv()` call. Definitely comes with some overhead, but will ensure that each test is isolated from other tests. It might be possible to write a single test fixture class, and then reuse it several times (via inheritance or template instantiation) so that you don't have to write this many times, but it's quite straight-forward and will save headaches in the future, I think.

This doesn't need to be done in this PR (since it's slightly a side comment, and the `setenv()` is from an earlier PR; please consider adding a TODO and/or filing an issue for this.

Reply:
Done.